frontend/src/constants.ts

```diff
@@ -1,19 +1,47 @@
 import { CardValue } from './types/WebSocket';
 
+export const TNG_URL = 'https://www.tngtech.com/en';
+
+// Special card values
+export const VOTE_COFFEE = 'coffee';
+export const VOTE_OBSERVER = 'observer';
+export const VOTE_NOTE_VOTED = 'not-voted';
+
+// Scales
 export const SCALES: { [id: string]: { name: string; values: Array<CardValue> } } = {
   FIBONACCI_SCALE: {
     name: 'Fibonacci',
-    values: ['coffee', '?', '0', '1', '2', '3', '5', '8', '13', '21', '34', '55', '89', '∞'],
+    values: [VOTE_COFFEE, '?', '0', '1', '2', '3', '5', '8', '13', '21', '34', '55', '89', '∞'],
   },
   COHEN_SCALE: {
     name: 'Cohen',
-    values: ['coffee', '?', '0', '0.5', '1', '2', '3', '5', '8', '13', '20', '40', '100', '∞'],
+    values: [VOTE_COFFEE, '?', '0', '0.5', '1', '2', '3', '5', '8', '13', '20', '40', '100', '∞'],
   },
   FIXED_RATIO_SCALE: {
     name: 'Fixed Ratio',
-    values: ['coffee', '?', '1', '2', '4', '8', '16', '32', '64', '128', '∞'],
+    values: [VOTE_COFFEE, '?', '1', '2', '4', '8', '16', '32', '64', '128', '∞'],
   },
-  SIZES_SCALE: { name: 'Sizes', values: ['coffee', '?', 'XS', 'S', 'M', 'L', 'XL', 'XXL', '∞'] },
+  SIZES_SCALE: { name: 'Sizes', values: [VOTE_COFFEE, '?', 'XS', 'S', 'M', 'L', 'XL', 'XXL', '∞'] },
 };
 
-export const TNG_URL = 'https://www.tngtech.com/en';
+// Text constants
+export const APP_NAME_FIRST = 'NEXT GENERATION';
```

Comment:
what do You think about a little more nesting:
const button = {
  login: 'login',
  ...
}
...

Reply:
Gleiches Argument wie an der anderen Stelle, ich weiß nicht, ob der Gewinn so groß ist. Aber ich kann die Konstanten gerne etwas sortieren.